### PR TITLE
Tweaks Bolt/Denied light to be a redder red, Changes Engineering alert light color to Orange

### PR DESCRIPTION
**Pull request by ghost**
## About The Pull Request
For the former half, people have always said the red looked too close to orange. Turns out they were really right; I made the red deeper and it already looks like a red Denied rather than a pinkish 'somethings wrong'. Feels much better.
(This does not affect the Sparks/Emag state, having the light look overloaded felt more fitting. Plus, changing the color underneath the sparking animation wasn't worth it lmao)

For the latter:
Within the first day of its merge, people asked why the doors were striped red.
Violently hot pink does not communicate 'engineering emergency', which was discussed in the PR that made them hot pink. It was agreed that orange would be better, and simply never done. The sprites also needed lots of general improvement.
I've used an actual common hexidecimal for OSHA's orange, and IMO it works quite well with a few other tweaks to the sprites: the dashed lines properly fit into all their doors' light strips, and the shading is tweaked slightly to blend more with the existing light colors as well.

Finally, not in the title, I sorted the Overlays files to all follow one structure because it was driving me nuts: 
Opening/Closing, Denied, Bolts, Power, Sparks, Emergency, Engineering, panels (sub-sorted too), Welded, Glass (optional), Sealed, Protected panels (sub-sorted too)

## How This Contributes To The Skyrat Roleplay Experience
Orange, as not only the engineering color, but OSHA's color for moderate warnings of hazards of any sort, make it more fitting for this: it's an Engineering emergency, and open areas may be hazardous to untrained crew.
(https://www.creativesafetysupply.com/articles/safety-colors/ | https://wp.stolaf.edu/ehs/labels-osha-color-coding/ | https://www.osha.gov/laws-regs/regulations/standardnumber/1910/1910.145AppA)

TL;DR, orange shows an engineering emergency much better than red-pink, which was mistaken for plain red

## Proof of Testing

<details>
<summary>Screenshots/Videos</summary>

Orange Engineering Lights
dreamseeker_YP4Urtby4y

Redder Bolt Lights
dreamseeker_Z6CqlwuvF4

</details>

## Changelog

:cl:
qol: The light color for Engineering Emergency Access has been changed to Orange, and for Bolts is now a redder red.
/:cl:

### CODE REVIEW

**Comment by lessthnthree** [comment]
In your own link it says orange is for a risk. The alert is incident management, which is pink. 

"It was agreed that orange would be better" wasn't a thing that happened either. If you go back and read the comments of the original PR, pink was the preferred option.

https://www2.gov.bc.ca/assets/gov/driving-and-transportation/transportation-infrastructure/engineering-standards-and-guidelines/technical-circulars/2012/t03-12.pdf

**Comment by ghost** [comment]
> In your own link it says orange is for a risk. The alert is incident management, which is pink.
> 
> https://www2.gov.bc.ca/assets/gov/driving-and-transportation/transportation-infrastructure/engineering-standards-and-guidelines/technical-circulars/2012/t03-12.pdf

This is the link for Roadways. This is a space station/workplace, not a road.
The alert is a major engineering **emergency**, not an incident. Incidents apply to minor outages, such as a single road. Which, again, doesn't apply here.
The given PDF is for only a specific organization (British Columbia Ministry of Transportation), explicitly says "fluorescent orange background will continue to be the primary choice of colour used", and is for **Incidents**, as mentioned above.
It does not apply to the situation on station, which is that Engineers need access to a **Hazard Area.**

Per my sources, "The warning category is for when there is a risk, but it is not as severe or immediate as when danger is used. The safety color associated with warning is orange or predominantly orange. As with the red, any lettering or symbols must be a contrasting color."
Red alert aptly applies to Danger. Warning, which isn't inherently lethal but should be treated with proper care, is a Hazard, or Risk.

And as for the agreement comment that you edited your comment for, it _did_ happen. It's in the comments, as well as on discord itself. Which I did, indeed, **read all of**. Orange was the preferred option besides a single random player agreeing with you.

**Comment by ghost** [comment]
Regardless, I've already cited my points, and discussed this with others. It will await review from maintainers, and I'll not argue further.

**Comment by press-mark** [comment]
idk orange makes more sense to me than hot pink.
just as long as it's distinct enough from emergency access

**Review by Ebin-Halcyon — APPROVED** [review]
Orange looks nice.

**Review by GoldenAlpharex — APPROVED** [review]
I already previously mentioned that I wanted orange for the engineering alert, and not pink. I thought that had been changed already in the original PR.
